nextflow.config

```diff
@@ -60,40 +89,53 @@ params {
     maps_cutoff_fdr            = 2
     maps_filter                = 'None'
     maps_model                 = 'pospoisson'
+    maps_3d_ext                = 'sig3Dinteractions.pe.txt'
     merge_map_py_source        = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/merge_map.py'
     feature_frag2bin_source    = 'https://raw.githubusercontent.com/ijuric/MAPS/91c9c360092b25a217d91b9ea07eba5dd2ac72f4/bin/utils/genomic_features_generator/scripts/feature_frag2bin.py'
     make_maps_runfile_source   = 'https://raw.githubusercontent.com/ijuric/MAPS/18c1a337f222130d7c5735d051614e2a253d5319/bin/MAPS/make_maps_runfile.py'
 
-    // Options: QC
-    skip_fastqc                = false
-    skip_plot_profile          = false
-    skip_igv                   = false
-    skip_multiqc               = false
-    skip_trackhub              = false
-    skip_enrichment            = true
+    // HiCDCPlus
+    hicdcplus_cutoff_fdr       = 0.05
+
+    // peakachu
+    peakachu_pretrained_url    = 'http://3dgenome.fsm.northwestern.edu/peakachu/HiCAR-models/HiCAR-peakachu-pretrained.'
```

Comment:
Is this URL correct?

Reply:
This url were provided by the peakachu authors. We have close collaboration with them and they generated the classifier for us.

Reply:
I am not familiar with this process. Is there any quick documentation for that. I will forward the doc to the authors. Another thing, there is another tool also called peakachu in bioconda. Will it be a concern?